lib/matplotlib/axes/_axes.py

```diff
@@ -1971,8 +2000,48 @@ def bar(self, left, height, width=0.8, bottom=None, **kwargs):
         See also
         --------
         barh: Plot a horizontal bar plot.
+
         """
         kwargs = cbook.normalize_kwargs(kwargs, mpatches._patch_alias_map)
+
+        matchers = [
```

Comment:
why not just
```
matchers = [
    lambda x, height, width=0.8, bottom=None, **kwargs:
        (False, x, height, width, bottom, kwargs),
    lambda left, height, width=0.8, bottom=None, **kwargs:
        (False, left, height, width, bottom, kwargs),
]
```
?

or did I miss a case?

Reply:
ah, that is better :sheep: